tools/test/travis-ci/doxy-spellchecker/en_phonet.dat

```diff
@@ -1,250 +0,0 @@
-#   phonetic_english.h - phonetic transformation rules for use with phonetic.c
-#   Copyright (C) 2000 Bjoern Jacke
-#
-#   This rule set is based on Lawrence Phillips original metaphone 
-#   algorithm with modifications made by Michael Kuhn in his
-#   C implantation, more modifications by Bjoern Jacke when
-#   converting the algorithm to a rule set and minor 
-#   touch ups by Kevin Atkinson
-#
-#   This library is free software; you can redistribute it and/or
-#   modify it under the terms of the GNU Lesser General Public
```

Comment:
@0xc0170 This would be why the test didn't catch the problem.

The license string is split across lines, and has a `#` in the middle of the linebreak...
This might be interesting to fix.

Reply:
@kegilbert Fyi.

It looks like this file isn't needed?

Reply:
If it's not needed we can remove, I had though Aspell was using it at the time but seems not to be.